.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Feature Request]: <title>"
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: is-related
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description:A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
there needs to be a space in between colon and text

Reply:
[Resolved](https://github.com/JaydenDev/Catalyst/pull/180/commits/42d0a058965bdabd04382cf88cdfc5ca7ff12a22)